continuationsPlugin/src/main/scala/continuations/DefDefTransforms.scala

```diff
@@ -25,6 +25,15 @@ import scala.collection.mutable.ListBuffer
 
 object DefDefTransforms extends TreesChecks:
 
+  def transformSuspendContinuation(tree: tpd.ValOrDefDef)(using Context): tpd.Tree =
```

Comment:
you may want to move `def transformContinuationWithSuspend` below this one as well?